src/nvim/shada.c

```diff
@@ -10,9 +5,6 @@
 #include <stdint.h>
 #include <inttypes.h>
 #include <errno.h>
-#ifdef HAVE_UNISTD_H
-# include <unistd.h>
-#endif
```

Comment:
This is already included for non-Windows systems in `os/os_defs.h`, which is included by `ascii.h`, which is included by `shada.c`.